src/main/drivers/exti.c

```diff
@@ -2,6 +2,7 @@
 #include <stdint.h>
 #include <string.h>
 
+#include "build/assert.h"
```

Comment:
`#include "build/assert.h"` should be after `#include "platform.h"` so that it can pick up the `#define USE_ASSERT` from the `target.h` file.